.github/steps/X-finish.md

```diff
@@ -1,23 +1,20 @@
-<!--
-  <<< Author notes: Finish >>>
-  Review what we learned, ask for feedback, provide next steps.
--->
-
 ## Finish 🏆
 
 _Congratulations friend, you've completed this course!_ 
 
 Here's a recap of all the tasks you've accomplished in your repository:
 
-- Enabled secret scanning on your repository
+- Enabled secret scanning if your repository has private or internal visibility
 - Committed a secret to the repository
 - Reviewed secrets that have been identified by secret scanning
 - Closed a secret scanning alert
-- Enabled secret scanning push protection to prevent secrets from being written to the repository
+- Enabled secret scanning push protection to prevent secrets from being written to the repository (required only for private or internal repositories)
 - Attempted to commit a secret, but had that commit stopped by push protection
-- Bypassed the push protection
+- Bypassed push protection
+
+It's important to note that secret scanning capabilities are available for free for all public repositories. Customers who want to enable secret scanning on private repos should find out more about [GitHub Advanced Security](https://docs.github.com/en/enterprise-cloud@latest/get-started/learning-about-github/about-github-advanced-security) or [Set up a trial of GitHub Advanced Security](https://docs.github.com/en/enterprise-cloud@latest/billing/managing-billing-for-github-advanced-security/setting-up-a-trial-of-github-advanced-security). 
```

Comment:
It seems worth including a link to the article about setting up a trial of GHAS as a call to action.